data/json/mapgen/bridges.json

```diff
@@ -125,16 +134,8 @@
         "ss=<UUUUUUUUUUUUUUUU<=ss",
         "ss####################ss",
         "ss####################ss",
-        "sssssssss######sssssssss",
-        "~~ssssssss####ssssssss~~",
-        "~~~~~~~~~ss##ss~~~~~~~~~",
-        "~~~~~~~~~~ssss~~~~~~~~~~",
-        "~~~~~~~~~~~~~~~~~~~~~~~~",
-        "~~~~~~~~~~~~~~~~~~~~~~~~",
-        "~~~~~~~~~~~~~~~~~~~~~~~~",
-        "~~~~~~~~~~~~~~~~~~~~~~~~",
-        "~~~~~~~~~~~~~~~~~~~~~~~~",
-        "~~~~~~~~~~~~~~~~~~~~~~~~",
+        "ssssssssssssssssssssssss",
```

Comment:
I kind of miss the partial bridge pylon here, but I can't see a good way to keep it.

Reply:
i missed it too, but i just wasn't sure how to keep it around either.